
arm_planner/Obstacle_msg[] obstacles
arm_planner/ControlPoint_msg[] control_points
arm_planner/ControlPoint_msg[] real_control_points
geometry_msgs/Point[] ee_traj
geometry_msgs/Point[] manipulation_area
geometry_msgs/Point cartesian_position
float64[] ee_orient
